dockstore-client/src/main/java/io/dockstore/client/cli/nested/ToolClient.java

```diff
@@ -810,7 +811,7 @@ public void handleInfo(String entryPath) {
                     description = "";
                 }
 
-                String author = container.getAuthor();
+                String author = container.getAuthors().isEmpty() ? "" : container.getAuthors().get(0).getName();
```

Comment:
If I'm reading this correctly, I didn't realize the CLI hadn't already been updated for the deprecation. Does this mean the 1.14 CLI will be broken against the 1.15 web service? If so, would it be better to not remove the deprecated fields from the web service until one more release?

Reply:
I think that yes, these two commands (updating and listing tools) will be broken. 
Technically, if we go one more release, I'd probably break apart the webservice PR into two parts 
1. deletion of the old authors fields in the models and DB can wait
2. for now, redirect the tests and queries

Reply:
thoughts @kathy-t ?

Reply:
Had a chat, we decided to make the webservice backwards compatible for existing 1.14 CLI users but merge most of the ui and cli changes so we don't have this problem in the 1.17 release